### DISCUSSION TITLE
Question 15: Why is the GenBankNumber (GenGenBankAccnNo_tab) field truncating and therefore not allowing the website to link out (see IRN 2631077)? [AWF- Mammals]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#986] [Q&A]
Description: This field is truncating the text such that the website does not work as an active link. We’d also like to have this link/field published to our online database- is this possible and what would that take?

**Comment by rondlg** [comment]
Response: We’ll look into thi Jo s. Might need a structural change.
- Adam to submit helpdesk / IT to submit jira/axiell tkt
- We will add this to the collections websites

**Comment by fmjjones** [comment]
Zendesk Ticket #69224
Hornbill ticket https://customer.hornbill.com/axiell/servicemanager/request/view/IN00206804/
